python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -689,18 +689,12 @@ router bgp {{ router_bgp.as }}
 {%             if peer_group.additional_paths.receive is arista.avd.defined(true) %}
       neighbor {{ peer_group.name }} additional-paths receive
 {%             endif %}
-{%             if peer_group.additional_paths.send.any is arista.avd.defined(true) %}
-      neighbor {{ peer_group.name }} additional-paths send any
-{%             elif peer_group.additional_paths.send.any is arista.avd.defined(false) %}
-      no neighbor {{ peer_group.name }} additional-paths send any
-{%             elif peer_group.additional_paths.send.backup is arista.avd.defined(true) %}
-      neighbor {{ peer_group.name }} additional-paths send backup
-{%             elif peer_group.additional_paths.send.ecmp is arista.avd.defined(true) %}
-      neighbor {{ peer_group.name }} additional-paths send ecmp
-{%             elif peer_group.additional_paths.send.ecmp_limit is arista.avd.defined %}
-      neighbor {{ peer_group.name }} additional-paths send ecmp limit {{ peer_group.additional_paths.send.ecmp_limit }}
-{%             elif peer_group.additional_paths.send.limit is arista.avd.defined %}
-      neighbor {{ peer_group.name }} additional-paths send limit {{ peer_group.additional_paths.send.limit }}
+{%             if peer_group.additional_paths.send_ecmp_limit is arista.avd.defined %}
+      neighbor {{ peer_group.name }} additional-paths send ecmp limit {{ peer_group.additional_paths.send_ecmp_limit }}
+{%             elif peer_group.additional_paths.send_limit is arista.avd.defined %}
+      neighbor {{ peer_group.name }} additional-paths send limit {{ peer_group.additional_paths.send_limit }}
+{%             elif peer_group.additional_paths.send is arista.avd.defined %}
```

Comment:
Add a check for `send` vlaue here, like -
`{%             if peer_group.additional_paths.send is arista.avd.defined and  peer_group.additional_paths.send == "ecmp" %}`

I think this all should come under a check - 
`{%             if peer_group.additional_paths.send is arista.avd.defined %}`
and the logic can be added under this to check for `ecmp` and `limit`

Reply:
same elsewhere

Reply:
Seconded!

Reply:
Added these checks.